try {

    // code...
    console.lo("working")
  
  } catch (err) {
  
    // error handling

    console.log("not working")
  
  }